install/docker/index.markdown

```diff
@@ -29,54 +29,35 @@ If you are running a recent version of Ubuntu (e.g. 14.04, 16.04) it can be as s
-
-Within the [MoveIt! Dockerhub page](https://hub.docker.com/r/moveit/moveit/) you'll find additional tag names you can use to help specify what exact ROS version you'd like to use. For example:
+## Running MoveIt! Containers
 
-    docker pull moveit/moveit:indigo-source
+To run a Debian-installed container of MoveIt! with graphics support:
 
-## Running MoveIt! Containers
+    wget https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/.docker/gui-docker gui-docker && chmod +x gui-docker
+    ./gui-docker -it --rm moveit/moveit:kinetic-release /bin/bash
 
-### New Users
+This will attempt to use nvidia-docker if hardware and drivers are available.
 
-For a debian-installed container of MoveIt:
+**Command-Line Only**
 
     docker run -it moveit/moveit:kinetic-release
 
-### Developers
+## MoveIt! Contrainer Types
 
-For a built-by-source container of MoveIt:
+There are many variants of the MoveIt! Docker available as documented [here](http://moveit.ros.org/documentation/contributing/continuous_integration/). For example, any of the three current distros work: [indigo, kinetic, lunar]. Over variations include:
 
-    docker run -it moveit/moveit:kinetic-source
+ - **moveit/moveit:kinetic-source** contains a full MoveIt! workspace downloaded and built to ~/ws_moveit/src. This container is useful for developers wanting to test or develop in a sandbox.
+ - **moveit/moveit:kinetic-release** builds ontop of the CI image, the full debian-based install of MoveIt! using apt-get.
+ - **moveit/moveit:kinetic-ci** an image optimized for running continuous integration with Travis
+ - **moveit/moveit:kinetic-ci-shadow** an image optimized for running continuous integration with Travis using the latest unreleased build of ROS
 
-Any of the three current distros work: [indigo, kinetic, lunar]
+## Advanced
 
 To make the source container more useful for developing and testing code, we recommend you install the following extra dependencies because the docker container is very bare-bones:
 
-    apt-get install less ssh bash-completion tree nano emacs
-
-## Build
+    apt-get install less ssh bash-completion tree nano
 
 MoveIt!'s docker containers are built automatically on dockerhub.com, but you can modify and build locally if desired with the following command:
 
-    cd kinetic/source
+    cd moveit/.docker/kinetic/source
     docker build -t moveit/moveit:kinetic-source .
-
-## Using GUI's with Docker
-
-These instructions are currently experiemental. For more details see the [ROS tutorial](http://wiki.ros.org/docker/Tutorials/GUI):
-
-    # This is not the safest way however, as you then compromise the access control to X server on your host
-    xhost +local:root # for the lazy and reckless
-
-    docker run -it --env="DISPLAY" --env="QT_X11_NO_MITSHM=1" --volume="/tmp/.X11-unix:/tmp/.X11-unix:rw" moveit/moveit:kinetic-source
-    export containerId=$(docker ps -l -q)
```

Comment:
Please tell the users where to execute this command since the previous command launches the docker bash and if the user executes the command inside docker image may get unknown command error